crowdin.py

```diff
@@ -446,11 +456,15 @@ def main():
     # Check if valid filepath for zip
     if options.zip_path:
         import_crowdin_for_android(options.zip_path, options)
-    validate_android_translations(options)
+    errors, warnings = validate_android_translations(options)
     if options.find_unused:
         print("\n\n")
         find_unused_keys(options)
 
 
+    # Return error code from the script if we have any errors
+    sys.exit(any(warnings.values()))
```

Comment:
Shouldn't this be `sys.exit(any(errors.values()))`? If there are warnings but not errors, the script should exit with a successful error code.

Reply:
Ooops, you're right! Had it backwards (since before it was also checking for both warnings and errors)